php.ini-development

```diff
@@ -356,6 +356,12 @@ disable_classes =
 ; http://php.net/realpath-cache-ttl
 ;realpath_cache_ttl = 120
 
+; Allows user to disable the "stat cache." By default php remembers the
+; results of the last call to stat() and lstat() and functions that call
+; it underneath.  Setting this to True disables that behaviour.
+; http://php.net/disable-stat-cache
+;disable_stat_cache = False
```

Comment:
While the stat cache is the default, I'd still recommend making the ini setting name positive, i.e. `enable_stat_cache = On/Off`.

Reply:
I picked disable as the default was enabled, if that makes sense?  But I can change it if it would be more consistent and have it default to On.

Reply:
And yes, it would be more consistent.  Changed!